### DISCUSSION TITLE
How do you run the bot?

### DISCUSSION THREAD

**Discussion by Lil-Pandaa** [danthe1st/Mee6-bypasser#31] [Questions] [answered]
Im new to discord.java. How do you run the bot?

**Comment by Lil-Pandaa** [comment]
Also I tried installing Maven to execute `mvn package` but it  just says 'mvn' is not recognized as an internal or external command, operable program or batch file.


Do you know any reasons it'd do that?

**Comment by danthe1st** [comment, marked as answer]
The library is named JDA, not discord.java.
In order to use maven (mvn, a java package manager just like npm for js), you need to [Download and install](https://maven.apache.org/download.cgi) it.

If you use IntelliJ, you can also press the Contr-Key twice and enter `mvn package`, in the text field that comes up, afterwards.

I have added build instructions in the README.
